main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+var (
+	tempFileName = "trivy-go-plugin-temp.json"
+)
+
+func main() {
+	trivyCommand := os.Args[1 : len(os.Args)-1]
+	tempFilePath := filepath.Join(os.TempDir(), tempFileName)
+	defer removeFile(tempFilePath)
+
+	cmdArgs := append(trivyCommand, "--format", "json", "--output", tempFilePath)
+	if err := exec.Command("trivy", cmdArgs...).Run(); err != nil {
+		log.Fatalf("failed to build report: %v", err)
+	}
```

Comment:
also, now we can check a case if `--output` file is already set, and just use it.